doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -55,19 +55,21 @@ Here it is assumed the notified/default FIWARE service maps the PostgreSQL schem
 [Top](#top)
 
 ####<a name="section1.2.2"></a>PostgreSQL tables naming conventions
-The name of these tables depends on the configured data model (see the [Configuration](#section2.1) section for more details):
+The name of these tables depend on the configured data model and analysis mode (see the [Configuration](#section2.1) section for more details):
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1057/commits/e70d1dfa29ba6dc4c20f2a649f115a985d585a36